crates/oxc_query/src/adapter.rs

```diff
@@ -136,11 +136,13 @@ impl<'a, 'b: 'a> trustfall::provider::Adapter<'a> for &'a Adapter<'b> {
                 property_name.as_ref(),
                 resolve_info,
             ),
-            "ObjectLiteral" => super::properties::resolve_object_literal_property(
-                contexts,
-                property_name.as_ref(),
-                resolve_info,
-            ),
+            "ObjectLiteralAST" | "ObjectLiteral" => {
```

Comment:
What's the difference between these two? 

Separately, why are we using strings here instead of enums?

Reply:
Difference: they are different items in the schema.graphql

why strings? That’s what trustfall’s api is.